src/Map.jsx

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from 'react';
-import ReactMapGL, { NavigationControl } from 'react-map-gl';
+import ReactMapGL, { NavigationControl, Popup } from 'react-map-gl';
 import {
   graphql,
   createFragmentContainer,
 } from 'react-relay';
 import propTypes from 'prop-types';
-import { MAP_STYLE, MAPBOX_ACCESS_TOKEN } from './config.json';
+import { MAP_STYLE, MAPBOX_ACCESS_TOKEN } from './config';
```

Comment:
why remove the `.json` part?

Reply:
I can't import json file into js (so i import variables from .js file), I think it's about my webpack configuration.